public/javascripts/gformFileUpload.js

```diff
@@ -28,6 +28,14 @@
     deleteLabel: {
       en: "Delete",
       cy: "Dileu"
+    },
+    uploadingFile: {
+      en: "Uploading file",
+      cy: "Uploading file"
+    },
+    hasBeenUploaded: {
+      en: "has been uploaded",
+      cy: "has been uploaded"
```

Comment:
I know we dont have to solve it now, but it would be good to initialize these labels from the messages properties file on page load, so we dont store translations in 2 different places (like we did in https://github.com/hmrc/gform-frontend/blob/master/app/uk/gov/hmrc/gform/views/govuk_wrapper.scala.html, but allow each page to define what it needs to load)

Reply:
Good call, I've made a note to fix this in upscan related work